source/module_base/test/complexarray_test.cpp

```diff
@@ -0,0 +1,207 @@
+#include"../complexarray.h"
+#include"gtest/gtest.h"
+
+/************************************************
+*  unit test of class ComplexArray
+***********************************************/
+
+/**
+* function "inline void ComplexArray::operator=(const std::complex < double> c)" 
+* has some error. 
+*
+* The declare of this function is in .h file, but the definition is in .cpp file,
+* and this would cause error when only include the .h file in this .cpp file.
+*
+* the definition of this function should be in .h file, 
+* or do not define as an "inline" function, but a normal function.
+*/
+
+
+/**
+* - Tested Function:
+*   - operator "=":
+*       - assign a complex to all elements of a ComplexArray
```

Comment:
The explanations here seems to be duplicated. I think the test name should satisfy.

Reply:
> The explanations here seems to be duplicated. I think the test name should satisfy.

the test name can reflect a general function, but may not reflect the detail tested functions, I still think this commands should be  saved.